lesson_10/test_homeworks10.py

```diff
@@ -0,0 +1,77 @@
+import unittest
+from homework10 import multiplication_table, longest_in_list, list_of_squares, sum_numbers_in_list
+
+
+class MultiplicationTable(unittest.TestCase):
+    
+    # Check maximum returned result
+    def test_multiplication_table_10(self):
+        actual = multiplication_table(10)
+        expected = "Task 1 --> 10 x 1 = 10\nTask 1 --> 10 x 2 = 20\n"
+        self.assertEqual(actual, expected)
+
+    # Check that returned result is a string
+    def test_multiplication_table_result_type(self):
+        actual = multiplication_table(3)
+        self.assertIsInstance(actual, str)
```

Comment:
```suggestion
        actual = multiplication_table(3)
        self.assertIsInstance(actual, str)
```
дуже слабка перевірка